mkdocs.yml

```diff
@@ -45,11 +45,15 @@ nav:
   - 'Obtaining DCR': 'getting-started/obtaining-dcr.md'
   - 'Using the Block Explorer': 'getting-started/using-the-block-explorer.md'
 - Governance:
+  - Politeia:
+    - 'Politeia': 'governance/politeia.md'
```

Comment:
Lets move the `politeia.md` file into the `politeia` directory so its URL matches its location in the nav menu. Need to set up a httpd redirect as well.

Reply:
Done.

Reply:
This one still needs the httpd redirect. Add a line to the bottom of `httpd.conf`.

```
Redirect /governance/politeia                                  /governance/politeia/politeia
```